README.md

```diff
@@ -1,3 +1,41 @@
 # Tier-Mark
-Tier module
+Tier module for HumbleBundle clone
+
+
+## Usage
+
+> To start server, from root directory in Terminal: "npm run server-dev"
```

Comment:
Do you plan to have your server open in development mode in production? That is, is the plan to use nodemon for your server exclusively, or do you intend to use node?

Reply:
Nah, I'll switch it over eventually. But we're not there yet. The README will be updated as I go.